lib/OpenQA/WebAPI/Plugin/AuditLog.pm

```diff
@@ -42,9 +42,9 @@ sub register {
         @table_events, @job_events, @jobgroup_events, @jobtemplate_events, @user_events,
         @asset_events, @iso_events, @worker_events,   @needle_events
     );
-    # filter out blacklisted events
-    my @blacklist = split / /, $app->config->{audit}{blacklist};
-    for my $e (@blacklist) {
+    # filter out events on blocklist; compatibility for old "blacklist" config entries
+    my @blocklist = split / /, $app->config->{audit}{blocklist} . ' ' . $app->config->{audit}{blacklist};
```

Comment:
Have you tested whether this works? Just because the old "blacklist" key doesn't appear anymore in ` lib/OpenQA/Setup.pm` but as far as I know we only read keys listed in the data structure defined there.

Besides, the repeated `$app->config->{audit}` and mixed concatenating and splitting don't look nice. I'd write it as `my @blocklist = split(/ /, $audit_config->{blocklist} || $app->config->{audit}{blacklist})` and would not allow people to specify both keys (as the old one is not supposed to be used anymore).

Reply:
Well I checked that a single line of code using `split / /, "$foo $bar"` works but haven't considered the concatenation warnings. Thanks for your proposal but I have now used a different approach by just doing the compatibility fix when reading the complete config so that we can keep the compatibility handling in one place.

Reply:
Good, that's better indeed.